openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -268,6 +268,9 @@ def create_skeleton_instance(
     instance_skeleton_data["representations"] = []
     instance_skeleton_data["representations"] += representations
 
+    if instance.data.get("stagingDir_persistent"):
+        instance_skeleton_data["stagingDir_persistent"] = True
```

Comment:
We should maybe set it all the time.
```suggestion
    persistent = instance.data.get("stagingDir_persistent") is True
    instance_skeleton_data["stagingDir_persistent"] = persistent
```